common/meta/raft-store/src/state_machine/sm_kv_api_impl.rs

```diff
@@ -21,15 +21,15 @@ use common_meta_types::MetaError;
 use common_meta_types::SeqV;
 use common_meta_types::TxnReply;
 use common_meta_types::TxnRequest;
-use common_meta_types::UpsertKVActionReply;
+use common_meta_types::UpsertKVReply;
 use common_meta_types::UpsertKVReq;
 use common_tracing::tracing;
 
 use crate::state_machine::StateMachine;
 
 #[async_trait::async_trait]
 impl KVApi for StateMachine {
-    async fn upsert_kv(&self, act: UpsertKVReq) -> Result<UpsertKVActionReply, MetaError> {
+    async fn upsert_kv(&self, act: UpsertKVReq) -> Result<UpsertKVReply, MetaError> {
```

Comment:
Does this rename will break the computable between metasrv and query?

Reply:
No. It is just a type alias and we have `tests/compat/test-compat` to ensure an old query talks happily to the latest metasrv.